scripts.py

```diff
@@ -1890,7 +1891,9 @@ def collections_with_uploader(self, collections):
             to use for that Collection.
         """
         for collection in collections:
-            uploader = MirrorUploader.for_collection(collection)
+            uploader = MirrorUploader.for_collection(
+                collection, ExternalIntegrationLink.COVERS
+            )
```

Comment:
Again, we now have different MirrorUploaders for different types of items. So  `ReplacementPolicy` needs to take two mirrors -- one for every type of thing that might be mirrored. It might be better to make `ReplacementPolicy` take a dictionary that maps "purpose" to an appropriate MirrorUploader.